packages/autocomplete-core/src/getPropGetters.ts

```diff
@@ -171,7 +171,7 @@ export function getPropGetters<
     const activeItem = getActiveItem(store.getState());
 
     const userAgent = props.environment.navigator?.userAgent;
```

Comment:
```suggestion
    const userAgent = props.environment.navigator?.userAgent ?? '';
```

this is also possible if we have babel set up correctly

Reply:
Short-circuiting is fine here though, no need for nullish coalescing. I've done it at the argument level on the next line.

Reply:
I meant that IMO it makes more sense to do it when the variable is created instead of used

```suggestion
    const userAgent = props.environment.navigator?.userAgent || '';
```

Reply:
Yeah I hesitated and refactored since this is the function that expects always a string. But honestly, it doesn't matter much 🙂

Reply:
Reverted the last commit.